### DISCUSSION TITLE
Feasibility to add Custom Date Filter

### DISCUSSION THREAD

**Discussion by DeekshaPremchandRao** [ghostfolio/ghostfolio#4145] [Ideas]
Can we add custom date filter, it would help to drill down data for weeks or months or a finite date range

**Comment by mvittox** [comment]
This would be a great feature

**Comment by vesely-david** [comment]
Adding +1! Especially if we could choose granularity to see the time series per year/month/week.

**Comment by fuzzysb** [comment]
+1 for this, would like to track over last Quarter to Date and 6 Month to date... but a custom date filter would be great

**Comment by Apollosport** [comment]
+1 from me as well. That would definitely help a lot

**Comment by Drakkarian** [comment]
+1 from me for that function as well
